datasources/src/main/java/org/opensearch/sql/datasources/transport/CreateUpdateDatasourceResponse.java

```diff
@@ -0,0 +1,12 @@
+package org.opensearch.sql.datasources.transport;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CreateUpdateDatasourceResponse {
```

Comment:
Add comments. this class is used by Create and Update Datasource response?

Reply:
Removed the class - respond in your other comment